src/Files.App/ViewModels/SidebarViewModel.cs

```diff
@@ -542,6 +543,26 @@ public async void EmptyRecycleBin(RoutedEventArgs e)
 			await RecycleBinHelpers.S_EmptyRecycleBin();
 		}
 
+		public async void OnRecycleBinChanged(object sender, EventArgs e)
+		{
+			var recycleBinLocationItem = SideBarItems
```

Comment:
I don't know if that Linq makes any sense. Wouldn't it be better if it was a single `foreach` loop? 🙂

Reply:
Done, even though I have no idea how to make it a one `foreach` loop only.